charts/airflow/values.yaml

```diff
@@ -791,6 +791,25 @@ workers:
     ##
     minAvailable: ""
 
+  ## configs for the worker Pods' readinessProbe probe
+  ##
+  readinessProbe:
+    enabled: true
+    initialDelaySeconds: 120
+    periodSeconds: 120
+    timeoutSeconds: 20
+    failureThreshold: 5
+
+  ## configs for the worker Pods' liveness probe
+  ##
+  livenessProbe:
+    enabled: true
+    initialDelaySeconds: 10
+    periodSeconds: 30
+    timeoutSeconds: 15
```

Comment:
Is 15 seconds enough to definitely have the `celery ...` command return?

Reply:
I tested in our environment and it was taking 4-5 seconds but we could increase this to be more resilient.